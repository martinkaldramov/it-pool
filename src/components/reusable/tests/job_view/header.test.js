import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../job_view/header';

describe('Header' () => {
  const wrap = shallow(<Header />);

  it('shoud render sccesfully', () => {
    expect(wrap).toMatchSnapshot();
  });
});