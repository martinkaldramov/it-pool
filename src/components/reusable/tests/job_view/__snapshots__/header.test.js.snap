// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header shoud render sccesfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="left-side-info"
>
          <p>
                     
                     
          </p>
</div>,
        <div
          className="right-side-info"
>
          <p>
                     
                     
                     
          </p>
</div>,
      ],
      "className": "job-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
             
             
</p>,
          "className": "left-side-info",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              " ",
              undefined,
              " ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            undefined,
            " ",
            undefined,
            " ",
            undefined,
          ],
          "type": "p",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
             
             
             
</p>,
          "className": "right-side-info",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              " ",
              undefined,
              " ",
              undefined,
              " ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            undefined,
            " ",
            undefined,
            " ",
            undefined,
            " ",
            undefined,
          ],
          "type": "p",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="left-side-info"
>
            <p>
                         
                         
            </p>
</div>,
          <div
            className="right-side-info"
>
            <p>
                         
                         
                         
            </p>
</div>,
        ],
        "className": "job-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
               
               
</p>,
            "className": "left-side-info",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                undefined,
                " ",
                undefined,
                " ",
                undefined,
              ],
            },
            "ref": null,
            "rendered": Array [
              undefined,
              " ",
              undefined,
              " ",
              undefined,
            ],
            "type": "p",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
               
               
               
</p>,
            "className": "right-side-info",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                undefined,
                " ",
                undefined,
                " ",
                undefined,
                " ",
                undefined,
              ],
            },
            "ref": null,
            "rendered": Array [
              undefined,
              " ",
              undefined,
              " ",
              undefined,
              " ",
              undefined,
            ],
            "type": "p",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
